pkg/parser/parser.peg

```diff
@@ -316,53 +316,82 @@ InlineAttributes <- "[" attrs:(GenericAttribute)* "]" {
     return types.NewAttributes(attrs.([]interface{}))								
 } 
 
-// Quoted Text attributes (attributes in front of marked up text).
-//
-// There are two forms, regular [role], and short-hand [.role#id].
-// They behave differently from each other and from other attribute lists.
-//
-// Document authors concerned with compatibility should stick to using
-// just alphanumerics, the hyphen, and the underscore, in these values,
-// and should also avoid whitespace.
-
-QuotedTextAttrs <- attrs:(QuotedTextRole / QuotedTextShortHand) {
-    return attrs, nil
-}
-
-// This is for [role].  Anything between the brackets is
-// considered, but it must not start with a # or dot.  Anything after a comma
-// (including the comma) is discarded.  (Future support for named attributes?)
-QuotedTextRole <- "[" role:QuotedTextRoleWord? "]" {
+// asciidoctor is inconsistent in handling quoted text attributes.
```

Comment:
let's avoid such comments on Asciidoctor, please.

Reply:
Ok. I wanted to relate that trying to match the "reference" implementation was .... challenging.

Reply:
Should I just delete the comment entirely?  Honestly you were probably the primary consumer for it -- although I guess if anyone else was surprised by the different handling for quoted text it might be good to note.

I've been trying to noodle through how asciidoctor handles attribute lists, and it *is* quite inconsistent.  There doesn't seem to be rhyme nor reason -- just different regular expressions I think.  Which, as I've noted, makes writing a reasonable grammar to emulate it's exact behavior somewhat .... challenging.  And ultimately I think not worth the effort -- the WG will probably clean up the grammar.

Reply:
I understand your intent and I appreciate your effort in explaining the challenges that you encountered and the solution you're proposing. I just don't feel comfortable with having negative comment on Asciidoctor here, that's all. Maybe you could comment with something more neutral?

Reply:
The commentary on this PR will hopefully suffice for now.  I don't know how to point out that our behavior is different, without talking about the other implementation.  And frankly the other implementation's behavior is probably a limitation that is unintentional (i.e. bug).